source/auth_n.rst

```diff
@@ -4526,11 +4502,9 @@ To challenge an SMS Factor, you send a request like this, with or without specif
 
 .. only:: java
 
-  .. todo::
+  To challenge an SMS Factor, call ``factor.createChallenge(challenge)`` with or without setting a message.
```

Comment:
add a `;` at the end of the java sentence. Please also clarify something like this: `... with or without setting a message. This operation will automatically cause the factor to be challenged, meaning that the user will receive a a code as soon as this operation is executed.`